### PR TITLE
title

### PR DESCRIPTION
**Pull request by Volodymyr-Boiko**

### CODE REVIEW

File: serenity/user_info/models.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+# class Question(models.Model):
+#     question_text = models.CharField(max_length=200)
+#     pub_date = models.DateTimeField('date published')
+#
+#
+# class Choice(models.Model):
+#     question = models.ForeignKey(Question, on_delete=models.CASCADE)
+#     choice_text = models.CharField(max_length=200)
+#     votes = models.IntegerField(default=0)
+
```
Comment (Volodymyr-Boiko) [inline serenity/user_info/models.py:4]: can you delete commented lines?

File: serenity/user_info/models.py [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+class User_info(models.Model):
+    name = models.CharField(max_length=50)
+    second_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    age = models.IntegerField()
+    sex = models.CharField(max_length=50)
+    picture = models.ImageField(max_length=50)
```
Comment (Volodymyr-Boiko) [inline serenity/user_info/models.py]: max_length for ImageField?? read the fucking manual (RTFM)

File: serenity/user_info/models.py [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+#     question = models.ForeignKey(Question, on_delete=models.CASCADE)
+#     choice_text = models.CharField(max_length=200)
+#     votes = models.IntegerField(default=0)
+
+
+class User_info(models.Model):
+    name = models.CharField(max_length=50)
+    second_name = models.CharField(max_length=50)
+    last_name = models.CharField(max_length=50)
+    age = models.IntegerField()
```
Comment (Volodymyr-Boiko) [inline serenity/user_info/models.py]: what happen, when I would try to save this field with the value below zero?

File: serenity/user_info/models.py [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+#
+# class Choice(models.Model):
+#     question = models.ForeignKey(Question, on_delete=models.CASCADE)
+#     choice_text = models.CharField(max_length=200)
+#     votes = models.IntegerField(default=0)
+
+
+class User_info(models.Model):
+    name = models.CharField(max_length=50)
+    second_name = models.CharField(max_length=50)
```
Comment (Volodymyr-Boiko) [inline serenity/user_info/models.py]: - second name (if second name exist);
  so - you have set arg: null=True (RTFM)

File: serenity/user_info/models.py (hunk 3 above)
Comment (zlyu88) [reply serenity/user_info/models.py]: which field do you mean?

Comment (Volodymyr-Boiko) [reply serenity/user_info/models.py]: age = models.IntegerField() 
does age can be  below zero?

**Pull request merged**